discover/reference/articles/04-form-fields.markdown

```diff
@@ -0,0 +1,66 @@
+# Form Field Types [](id=form-field-types)
+
+A form without fields is no form at all. To meet your form-building needs,
+Liferay Forms gives you many useful and highly configurable field types.
+
+![Figure 1: Out of the box form field types.](../images/forms-field-types.png)
+
+Form Text
+: Users do not enter data into Form Text fields. The form creator enters text
+that form users see displayed on the form. This is useful for longer
+instructions.
+
+![Figure 2: A form text field.](../images/forms-form-text.png)
+
+Text Field : Users enter text into these fields. For example, a Full Name field
+will often be a text field. By default, a text field keeps all input on a single
+line of text. To accommodate longer responses, choose the multiline setting when
+configuring the text field. Place limits on the text users can enter (numbers
+from 1-10, or email addresses, for example) by using the Text field's validation
+options.
+
+![Figure 3: A multiline text form.](../images/forms-multiline.png)
+
+Select from List
+: Users select one (or more, if configured to allow it) options from a dropdown
+list of choices. Choices can be entered manually or automatically populated by
+a data provider. For example, a Country of Residence field would probably be a
+select from list field populated by a Countries of the World data provider. 
+
+![Figure 4: A select from list field.](../images/forms-select-list.png)
+
+Single Selection
+: Using a radio button, users select one option from a list of options displayed
+on the form. 
+
+![Figure 5: A single selection field.](../images/forms-single-selection.png)
+
+Date
+: Users select a date using a datepicker. For example, a Birth Date field would
+probably use the Date field type.
+
+![Figure 6: A date field.](../images/forms-date.png)
+
+Multiple Selection/Single Checkbox
+: Users select one or more options from check boxes (or switchers, if
+configured).
+
+![Figure 7: A multiple selection field using a switcher.](../images/forms-switcher.png)
+
+<!-- Unreleased fields
+Grid
+: Using radio buttons, users select from options laid out in rows and columns.
+One selection can be made per row. This is useful when the same response metric
+is needed for multiple questions. For example, a product survey form might ask
+users to rate a list of their product's characteristics as Wonderful, Pretty
+Good, Not So Good, or Awful.
+
+![Figure 8: A grid field.](../images/forms-grid.png)
+
+Numeric
+: Users enter numeric data (integers or decimals) into numeric fields.
+Non-number input is not accepted. For example, configure a numeric field that
+accepts integers to ask users how many pets they have.
+
+![Figure 9: A numeric field.](../images/forms-numeric.png)
+-->
```

Comment:
@rbohl, these fields aren't released yet, but they appear in the **Out of the box form field types** image, thus I believe that would be better to sync the content of these two parts to avoid misunderstandings. What do you think?

Reply:
Forget the comment below, because I've just seen your last commit which has updated the mentioned image.